{{ cookiecutter.project_slug }}/flows/index.yml

```diff
@@ -9,14 +9,7 @@ pods:
     parallel: $JINA_PARALLEL
     timeout_ready: 600000
     read_only: true
-  chunk_indexer:
-    uses: pods/chunk.yml
-    shards: $JINA_SHARDS
-    separated_workspace: true
   doc_indexer:
     uses: pods/doc.yml
```

Comment:
It should still be `chunk.yml`. Or rename the file.

Reply:
Done